buildSrc/src/main/groovy/org/robolectric/gradle/SpotlessPlugin.groovy

```diff
@@ -0,0 +1,40 @@
+package org.robolectric.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class SpotlessPlugin implements Plugin<Project> {
+    void apply(Project project) {
+        project.getPlugins().apply('com.diffplug.spotless')
+
+        project.spotless {
+            kotlin {
+                // Add configurations for Kotlin files
+                target '**/*.kt'
+                ktfmt('0.42').googleStyle()
+            }
+            java {
```

Comment:
Don't we need to keep the `googleJavaFormat` here?

Reply:
> Maybe not. @Devashishbasu Maybe we can remove google's formatting configuration directly. I can accept removing it.

Done